pkgs/applications/networking/instant-messengers/teams/default.nix

```diff
@@ -103,7 +103,8 @@ let
       mv share $out/share
 
       substituteInPlace $out/share/applications/teams.desktop \
-        --replace /usr/bin/ ""
+        --replace /usr/bin/ "" \
+        --replace "Icon=teams" "Icon=$out/share/pixmaps/teams.png"
```

Comment:
This would break theming support, a better option would be to link the image into the hicolor icon theme.

Like this, for example: 
`ln -s $out/share/pixmaps/teams.png $out/share/icons/hicolor/512x512/apps/teams.png`

Reply:
Moved the icon to the hicolor path.

Reply:
Moving the icon into hicolor is so that it can be detected by `Icon=teams` as the default icon, so this line `--replace "Icon=teams" "Icon=$out/share/icons/hicolor/512x512/apps/teams.png"` is no longer needed and would in fact hinder the theming of the icon.